### PR TITLE
Added links to the C/C++ properties reference

### PR DESCRIPTION
**Pull request by spacemonkd**
Added the link to C/C++ properties JSON reference as requested in issue
[#5875 ](https://github.com/microsoft/vscode-cpptools/issues/5875)

### CODE REVIEW

File: Extension/ui/settings.html [hunk 1]
```diff
@@ -552,7 +556,7 @@
                 <option value="c11">c11</option>
                 <option value="c99">c99</option>
                 <option value="c89">c89</option>
-                <option value="gnu18">gnu18</option>
+                <option value="gnu17">gnu18</option>
```
Comment (sean-mcmanus) [inline Extension/ui/settings.html]: Why was this changed? Can you change this back? We decided to just expose c18/gnu18 in settings, although we'll accept -std=c17/gnu17 via the compilerArgs (which will convert it to c18 internally).

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/ui/settings.html (hunk 1 above)
Comment (spacemonkd) [reply Extension/ui/settings.html]: Okk

**Comment by spacemonkd** [comment]
Change made and updated. Kindly check @sean-mcmanus

**Comment by spacemonkd** [comment]
If time permits could you check and merge the current branch @sean-mcmanus 😅

Thank you

**Comment by spacemonkd** [comment]
@sean-mcmanus sir could you please review the code and merge it if no more changes are required or inform me of further changes.

**Comment by spacemonkd** [comment]
Can we close this PR?😅😅 @sean-mcmanus

File: Extension/ui/settings.html [hunk 2]
```diff
@@ -423,6 +423,10 @@
 
         <span data-loc-id="switch.to.json">Switch to the <a href="command:C_Cpp.ConfigurationEditJSON" title="Edit configurations in JSON file" data-loc-id-title="edit.configurations.in.json">c_cpp_properties.json</a> file by clicking on the file link or using the command:</span><br> 
         <code data-loc-id="edit.configurations.json">C/C++: Edit configurations (JSON)</code>
+
+        <div style="height: 30px; display: block"></div>
+
+        <span data-loc-id="check.the.schema">Check out <a href="https://code.visualstudio.com/docs/cpp/c-cpp-properties-schema-reference" title="Properties Schema Reference" >C/C++ Properties Schema Reference</a>. Click the link to learn more about the various properties.</span><br>
```
Comment (sean-mcmanus) [inline Extension/ui/settings.html]: @Colengms @michelleangela Any opinions on the wording. I was thinking it would be better to be shorter, like "See the C/C++ Properties Reference to learn more about the various properties.".

Comment (spacemonkd) [reply Extension/ui/settings.html]: Okk will shorten it😊

Comment (spacemonkd) [reply Extension/ui/settings.html]: How about "Check out the C/C++ properties reference **here**"

@sean-mcmanus

Comment (bobbrow) [reply Extension/ui/settings.html]: Please also make sure to use the localization tag for "title" in the href element.

Comment (sean-mcmanus) [reply Extension/ui/settings.html]: "Check out" seems too informal to be used in UI text. Also, is the usage of "here" necessary if the link is the "C/C++ properties reference"?

Comment (michelleangela) [reply Extension/ui/settings.html]: Please change the wording to the following:

Learn more about the C/C++ properties by going to [C/C++ Properties Schema Reference](https://code.visualstudio.com/docs/cpp/c-cpp-properties-schema-reference).

Comment (sean-mcmanus) [reply Extension/ui/settings.html]: @michelleangela That seems fine to me.

Comment (spacemonkd) [reply Extension/ui/settings.html]: Okayy

**Review by michelleangela — APPROVED** [review]

**Review by sean-mcmanus — APPROVED** [review]
